src/lib.rs

```diff
@@ -62,14 +62,29 @@ fn get_num_cpus() -> usize {
     cpus as usize
 }
 
-#[cfg(unix)]
+#[cfg(target_os= "linux")]
 fn get_num_cpus() -> usize {
+    //to-do: replace with libc::_SC_NPROCESSORS_ONLN once available
     unsafe {
-        libc::sysconf(84  /*libc::_SC_NPROCESSORS_ONLN*/) as usize
+        libc::sysconf(84) as usize
+    }
+}
+
+#[cfg(target_os="macos")]
+fn get_num_cpus() -> usize {
+    //to-do: replace with libc::_SC_NPROCESSORS_ONLN once available
+    unsafe {
+        libc::sysconf(58) as usize
     }
 }
 
 #[test]
-fn it_works() {
+fn lower_bound() {
     assert!(get() > 0);
 }
+
+
+#[test]
+fn upper_bound() {
+    assert!(get() < 236,451);
```

Comment:
You can't actually use commas in number literals like this. You *can* however use underscores. I'll just merge and fix myself to `236_451`.